templates/errors/404.html

```diff
@@ -0,0 +1,17 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<div class="page-wrap d-flex flex-row align-items-center">
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-12 text-center">
+                <span class="display-1 d-block">404</span>
+                <div class="mb-4 lead">The resourse you are looking for was not found.</div>
+                <div class="mb-4 lead">{{error_msg}}</div>
```

Comment:
If this page is meant only for HTTP/404, the error message is already there one line above. What is the purpose of this one?

Reply:
the idea was to have an option for additional message, but I am removing this for now and will add it if needed in the future